schema.sql

```diff
@@ -69,7 +69,11 @@ CREATE TABLE genotypes (
        quality TEXT,
 
        -- Cycledash-derived data
-       "annotations:gene_names" TEXT,
+       -- These are from the varcode annotation worker
+       "annotations:gene_name" TEXT,
```

Comment:
Nope -- this annotation only picks an effect in a single gene. So we won't be having multiple genes annotations any more.